pkg/jx/cmd/common_install.go

```diff
@@ -196,6 +199,106 @@ func (o *CommonOptions) downloadFile(clientURL string, fullPath string) error {
 	return nil
 }
 
+func (o *CommonOptions) installOrUpdateBinary(binary string, gitHubOrganization string, downloadUrlTemplate string, version string, skipPathScan bool) error {
+	binaries := map[string]string{}
+	configDir, err := util.ConfigDir()
+	if err != nil {
+		return err
+	}
+	binariesConfiguration := configDir + "/binaries.yml"
```

Comment:
Please could you use `filepath.Join` here is more portable?

Reply:
Good point. I will change it :)